guides/7-jlp/2-How-JLP-Works.md

```diff
@@ -33,6 +33,12 @@ JLP can also be sold via Jupiter Swap. Simply swap JLP for your desired asset. T
 The best way to purchase or exit JLP is always via [Jupiter Swap](https://jup.ag/swap/USDC-JLP).
 :::
 
+:::info
+When using Jupiter Swap or programs that integrate Jupiter Swap, the minting and burning of JLP tokens occur automatically. If you're swapping any token for JLP, the system will mint JLP at the virtual price, which increases the overall JLP supply. Conversely, when you swap JLP for other tokens, the system burns the amount of JLP you provided, thereby decreasing the JLP supply. This automatic minting and burning functionality is integrated directly into the Jupiter Swap program.
```

Comment:
Jupiter Swap will find the best route that is either mint/burn the token or swap thru other DEXes that have the best route

Reply:
I think this portion is worth having its own subheader under "Add/Remove Liquidity into JLP". Will propose an edit

Reply:
Oh ya, @wowmiir helped added this here: https://github.com/jup-ag/space-station/pull/499/commits/be7855e1ef6a60108f8dee3d225c09b2ceae5a66

Thanks @wowmiir !